### DISCUSSION TITLE
How to know if a related table has records

### DISCUSSION THREAD

**Discussion by MML1357** [prisma/prisma#7921] [Q&A] [answered]
Hi. Having this models

```prisma
model event {
  id     Int     @id @default(autoincrement())
  name   String? @db.VarChar
  active Boolean @default(true)
  area   area?   @relation(fields: [areaId], references: [id])
  areaId Int?
}

model area {
  id    Int     @id @default(autoincrement())
  name  String? @db.VarChar
  event event[]
}
```

How can I get de `areas` who has records in `event`?

**Comment by ryands17** [comment, marked as answer]
@MML1357 👋 

You can do it like this:

```ts
await prisma.area.findMany({ where: { event: { some: {} } } })
```

You can read more about relation queries [here](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#relation-filters).

**Reply by MML1357** [reply]
Thanks. That's what I need

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
